__tests__/authors.test.js

```diff
@@ -0,0 +1,30 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('authors routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('should return list of authors', async () => {
+    const res = await request(app).get('/authors');
+    expect(res.body.length).toEqual(6);
```

Comment:
ideally the test would be a little more robust, as soon as another book is added to the database this test will fail despite the fact it the route still might be working correctly. you could check just the first book to make sure the object is the right shape